site/content/docs/user/configuration.md

```diff
@@ -308,6 +308,9 @@ With the installation of the Docker Desktop Application, whether it is on macOs,
 
 You may also want to see the [Ingress Guide].
 
+> **NOTE**: If you're running Kind on a remote host and need to send traffic to Kind node
+> IPs from a non-remote host, you need to configure port-mapping.
```

Comment:
Er, port forwarding is generally expected to be used in this guide, even in the local case (especially considering usage like docker desktop)

Reply:
Hi Ben, could you clarify? I don't understand what you mean.

Reply:
... I'm sorry, I read this as the ingress guide.

----

We have a comment above that outlines when you can not use port mappings to reach services listening on the node already, I'm not sure what case this is trying to cover?

What is "non-remote host", wouldn't that just be the host machine?

You need a port mapping to reach nodePort / hostPort services UNLESS you're on the same host as the kind node and actually local to the nodes / not using a VM solution like docker desktop.

(You can also avoid this by sending traffic from another container in all cases)

Reply:
> What is "non-remote host", wouldn't that just be the host machine?

I am running Kind on a cloud VM, the non-remote host is the machine I am using to access the cloud VM.

Reply:
I think I agree with Ben here that "non-remote host" is a little confusing and would be better stated as "local host machine" or "your local machine". Or maybe "from a different host than where kind is running"?

Reply:
I love this suggestion "from a different host than where kind is running". 

I'll update it, thanks for the feedback, @stmcginnis.